packages/common.vm/tools/vm.common/vm.common.psm1

```diff
@@ -157,7 +157,7 @@ function VM-Check-Reboot {
     )
     try {
         if (Test-PendingReboot){
-            VM-Write-Log "ERROR" "Host must be rebooted before continuing install of $package.`n"
+            VM-Write-Log "ERROR" "Host must be rebooted before continuing installation of $package.`n"
```

Comment:
Something weird happen while rebasing as I am the author of the commit and you the commiter (you should get authorship) and thisthis change is already in main. But I think merging shouldn't break anything